test/test_torch.py

```diff
@@ -17588,11 +17592,11 @@ def _helper_test_igamma(self, loglo, loghi, device, dtype):
         ]
         half_prec = dtype in [torch.bfloat16, torch.float16]
         for input0, input1 in inputs:
-            actual = torch.igamma(input0, input1)
+            actual = torch_fcn(input0, input1)
```

Comment:
Add a test with the out= variant here, too, and compare out to actual. The out tensor can be empty_like actual.